frontend/docsettings.lua

```diff
@@ -360,6 +437,19 @@ end
 --- Updates sdr location for file rename/copy/move/delete operations.
 function DocSettings:updateLocation(doc_path, new_doc_path, copy)
     local doc_settings, new_sidecar_dir
+    
+    if G_reader_settings:readSetting("document_metadata_folder") == "hash" then
```

Comment:
You've missed `cache_file_path` in this codeflow.
Can we avoid code duplicating?